scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -4,6 +4,7 @@ import mill._
 import mill.scalalib._
 import mill.api.PathRef
 import mill.define.ModuleRef
+import os.{CommandResult, Path}
```

Comment:
Generally we should avoid importing these and instead use them fully qualified as `os.CommandResult` and `os.Path`